lib/icinga/downtime.cpp

```diff
@@ -317,9 +317,10 @@ void Downtime::RemoveDowntime(const String& id, bool cancelled, bool expired, co
 	String config_owner = downtime->GetConfigOwner();
 
 	if (!config_owner.IsEmpty() && !expired) {
-		Log(LogWarning, "Downtime")
-			<< "Cannot remove downtime '" << downtime->GetName() << "'. It is owned by scheduled downtime object '" << config_owner << "'";
-		return;
+		String errorMessage = "Cannot remove downtime '" + downtime->GetName() +
+				"'. It is owned by scheduled downtime object '" + config_owner + "'";
```

Comment:
What's this variable for? If just for readability: OK, then please std::move() it...

Reply:
No, this is not for readability but because you cannot pass string concatenation as parameter directly und moving it brings here nothing because.

Reply:
Where's your last sentence from?

Reply:
from `Clang-Tidy`.

Reply:
> you cannot pass string concatenation as parameter directly

Why not?

Reply:
oh yes, my mistake!